src/redux/reducers/reducer.js

```diff
@@ -0,0 +1,18 @@
+const reducer = (state, action) => {
+  switch (action.type) {
+    case 'update/users':
```

Comment:
action types are usually named [like this](https://www.valentinog.com/blog/redux/#react-redux-tutorial-getting-to-know-redux-actions-and-named-constants) (defined as constants using UPPER_CASE)